src/main/webapp/src/components/lyrics.js

```diff
@@ -1,15 +1,77 @@
 import React from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/styles";
+import { lighten } from '@material-ui/core/styles';
 import Typography from "@material-ui/core/Typography";
+import LinearProgress from '@material-ui/core/LinearProgress';
+import AudioPlayer from 'material-ui-audio-player';
+import axios from "axios";
+
+const styles = () => ({
+  root: {
+    minWidth: "300px",
+    display: "flex",
+    flexDirection: "column"
+  },
+  audio: {
+    margin: "20px 0 20px 0",
+  }
+});
+
+const StyledLinearProgress = withStyles({
+  root: {
+    backgroundColor: lighten('#000000', 0.5), 
+  },
+  bar: {
+    background: '#000000',
+  },
+})(LinearProgress);
 
 /**
  * Displays search component for song lyrics.
  */
 class Lyrics extends React.Component {
   constructor(props) {
     super(props);
+
+    this.state = {
+      speechUrl: undefined,
+      isLoading: false,
+      error: null,
```

Comment:
Why are we initializing `speechUrl` as `undefined` and `error` as `null`? Shouldn't we use the same value?

Reply:
`undefined` means that we just haven't defined/received `speechUrl` yet, but `null` means there's no error

Reply:
Gotcha